draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -176,7 +176,7 @@ defined in [@!RFC1982].  Otherwise, catalog consumers might not notice
 updates to the catalog zone's contents.
 
 There is no requirement to be able to query the catalog zone via recursive nameservers.
-Catalog consumers SHOULD ignore NS record at apex.
+Catalog consumers MUST ignore NS record at apex.
 However, at least one is still required so that catalog zones are syntactically correct DNS zones.
 A single NS RR with a NSDNAME field containing the absolute name "invalid." is RECOMMENDED [@!RFC2606;@!RFC6761].
```

Comment:
```suggestion
Therefore NS RRs at the apex are meaningless to the catalog consumer and MUST like any other meaningless RRs be ignored.
However, to accommodate name server software that refuses to load zones without an NS RRset at a zone's apex, we RECOMMEND to have a single NS RR with a NSNAME field  containing the absolute name "invalid." [@!RFC2606;@!RFC6761]
```

Reply:
Ah, but #53 fixed this by simplifying the requirement to a central "must ignore whatever they don't care about", so I think we can resolve this conversation.

Reply:
@Habbie Yes, I now also prefer removing the sentence actually. So I prefer PR #53  and would like to merge it sooner than later, but #53 itself does still have a nit I'd like your opinion on.